src/univariate/continuous/generalizedpareto.jl

```diff
@@ -48,7 +49,8 @@ function GeneralizedPareto(μ::Integer, σ::Integer, ξ::Integer)
     GeneralizedPareto(float(μ), float(σ), float(ξ))
 end
 GeneralizedPareto(σ::T, ξ::Real) where {T <: Real} = GeneralizedPareto(zero(T), σ, ξ)
```

Comment:
BTW no need for a type parameter here:
```suggestion
GeneralizedPareto(σ::Real, ξ::Real) = GeneralizedPareto(zero(σ), σ, ξ)
```